### PR TITLE
Optional ubuntu interface

### PR DESCRIPTION
**Pull request by aknot242**

### CODE REVIEW

File: ansible/inventory.yml [hunk 1]
```diff
@@ -7,8 +9,11 @@ all:
       hosts:
         10.1.1.5:
           ansible_connection: ssh
+          ansible_user: "ubuntu"
```
Comment (mjmenger) [inline ansible/inventory.yml:12]: are all of the formatting and variable additions reflected in https://github.com/mjmenger/terraform-aws-bigip-setup/blob/master/hostvars_template.yml

**Comment by aknot242** [comment]
@shiftcommathree would you be willing to test and review this PR? We want to get the PRs closed soon so we can merge back to upstream.

**Review by shiftcommathree — DISMISSED** [review]
This change has been functionally tested and is working.

**Comment by aknot242** [comment]
I fixed a merge conflict. Can one of you please review and approve? Thanks!

**Review by mjmenger — APPROVED** [review]

**Pull request merged**